server/py/framework/db/sqldb/models.py

```diff
@@ -349,6 +349,7 @@ class Run(Base, mlrun.utils.db.HasStruct):
         # TODO: change to JSON, see mlrun/common/schemas/function.py::FunctionState for reasoning
         body = Column(SQLTypesUtil.blob())
         start_time = Column(SQLTypesUtil.timestamp())
+        end_time = Column(SQLTypesUtil.timestamp())
```

Comment:
why not datetime? I know that other fields in the table use it, but since it's new one and we are planning to move all things to datetime (because the latest year supported by ts is 2038 iirc)

Reply:
That is correct - https://iguazio.atlassian.net/browse/ML-6921